VAMobile/src/utils/secureMessaging.tsx

```diff
@@ -510,22 +505,19 @@ export const getLinkifiedText = (
         NUMBERS_ONLY_REGEX_EXP.test(text) &&
         NUMBERS_ONLY_REGEX_EXP.test(nextText)
       ) {
-        textReconstructedBody.pop()
-        textReconstructedBody.pop()
-        textReconstructedBody.push(
-          <TouchableWithoutFeedback
-            onPress={() => {
-              launchExternalLink('tel:' + previousText + text + nextText)
-            }}
-            accessibilityRole="link"
-            accessible={true}
-            accessibilityLabel={getNumberAccessibilityLabelFromString(previousText + text + nextText)}
-            accessibilityHint={t('openInPhoneMessaging.a11yHint')}>
-            <TextView variant="MobileBodyLink">{previousText + ' ' + text + ' ' + nextText}</TextView>
-          </TouchableWithoutFeedback>,
+        const linkProps: LinkProps = {
+          text: previousText + ' ' + text + ' ' + nextText,
+          type: 'call',
+          phoneNumber: previousText + text + nextText,
+          a11yLabel: getNumberAccessibilityLabelFromString(previousText + text + nextText),
+          a11yHint: t('openInPhoneMessaging.a11yHint'),
+        }
+
+        linkCollection.push(
+          <Box mb={theme.dimensions.standardMarginBetween}>
+            <Link icon={{ preventScaling: true }} {...linkProps} />
```

Comment:
Please use LinkWithAnalytics instead of Link for all of these. That way we get analytics for free (props are the same)

Reply:
updated